tests/benchmarks/peripheral_load/src/common.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef PERIPHERAL_STRESS_TEST_H
+#define PERIPHERAL_STRESS_TEST_H
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
```

Comment:
this header doesn't need these includes

Reply:
it's the only place where these includes exist.
Test doesn't compile when I comment it out.